src/lib/Hydra/Helper/Nix.pm

```diff
@@ -589,4 +591,13 @@ sub isLocalStore {
 }
 
 
+sub constructRunCommandLogPath {
+    my ($filename) = @_;
```

Comment:
I'm missing a test for this sub in Hydra/Helper/Nix.t. I think we should also test what happens if the UUID is empty, and possibly a brief check of what we expect it to look like, similar to https://github.com/NixOS/hydra/pull/1137/files#diff-56076a02b6158367f4b32f3aa1acb339ed87a357e5223503eb2e3f3a2e427daeR26. Also note my comment in Root.pm about taking the record here.

Reply:
I've added a test for this, but unsure what you mean about testing an empty UUID, now that we've switched to giving the `RunCommandLog` class to the helper function.